src/app/qgssourcefieldsproperties.cpp

```diff
@@ -286,7 +291,7 @@ bool QgsSourceFieldsProperties::addAttribute( const QgsField &field )
 
 void QgsSourceFieldsProperties::apply()
 {
-  QSet<QString> excludeAttributesWMS, excludeAttributesWFS;
+  QSet<QString> excludeAttributesWMS, excludeAttributesWFS, includeTemporalFilters;
```

Comment:
includeTemporalFilters is unused, right?

Reply:
```suggestion
  QSet<QString> excludeAttributesWMS, excludeAttributesWFS;
```
yes: leftover